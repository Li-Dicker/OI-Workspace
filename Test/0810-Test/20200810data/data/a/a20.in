10
196
41549 65410 4429 24333 46129 75569 93061 14454 8017 69131 86837 87621 57337 86417 20982 85357 24899 38377 69456 25387 52421 49982 34775 36015 8996 65302 13760 13341 12851 35228 82087 68914 11432 82821 42196 15375 33304 16092 50480 91946 26778 60305 99418 4538 80480 42837 33677 2293 24874 78257 94189 50258 14770 82305 78572 79643 65049 34108 1824 34239 27149 77394 85933 73453 22863 42934 71623 82302 22276 30236 11126 17144 30340 77704 94851 32289 7440 68428 40522 94383 88514 66615 38209 46626 11813 49147 85454 75932 92483 46840 25306 11350 44307 28783 67030 96053 7781 79472 97846 71295 6070 98114 79046 82607 45105 73841 28148 8137 22328 96540 26530 2380 57634 47048 44239 72248 49831 40496 91995 91187 17443 96976 47418 50616 20485 62018 16906 25189 16200 42529 11287 48289 95783 47298 33473 33443 3732 91579 79140 18511 18889 42454 60321 26335 90445 10616 78130 61749 49796 90426 92366 4725 56454 2802 23398 46657 92072 67805 19120 95119 48971 98595 46489 26717 17207 54505 38930 89937 62015 80067 38985 61170 48463 70610 66433 8699 94297 99237 22125 43479 9553 85544 59284 87913 68445 88887 21516 43256 54751 87693 29053 97694 56108 68376 61107 5282 
35370 38385 38120 2689 35045 43659 836 23990 38423 19155 18426 28352 8837 22843 33619 27893 36077 43940 41467 27532 7808 9858 43554 32688 21356 35539 10455 49271 30985 31465 37638 26483 22747 11294 22352 49634 20440 18040 20000 49502 23854 29378 47165 48905 15419 31405 20304 12097 31972 12179 2740 49920 45795 4878 17739 18188 8843 7754 38600 36293 44122 38433 32124 8333 14064 46633 47991 23475 24981 24585 9802 12535 1327 40523 39623 28736 23566 20088 12188 45350 37319 16742 25933 37763 9487 14214 21687 6657 6985 14792 21680 7694 31601 33217 17869 3633 36916 29434 30562 4061 36292 34868 16106 40530 32568 4190 10665 6660 47100 8432 42052 23353 41459 38871 39690 2957 1326 10138 8188 40345 20319 48231 8354 4684 31359 43936 20742 16718 31666 35404 36991 47803 43671 43903 39280 6084 27624 14683 2042 40448 20167 31767 44392 30788 7171 15970 27493 10539 37890 25945 28309 36364 1941 45138 20720 38845 42406 39433 10406 25362 49375 26111 3154 6594 17510 2913 14186 2171 34103 8068 47549 43749 1776 26863 37872 21290 8262 4119 35219 8287 18499 41835 3772 4197 8085 11255 30852 49689 10609 19873 43804 25964 14544 47613 16326 14797 
198
18683 37713 8160 96012 3630 18663 83208 28678 88572 31662 30603 99466 75598 70760 99712 43999 42296 99203 76701 4817 97390 48424 22536 87103 32055 60605 70417 15811 75338 71506 40821 66837 89793 82221 50566 39091 69582 67039 72171 52643 75377 3749 73736 37446 12913 93090 93274 32904 69383 76266 22925 50330 69060 18031 50258 61431 88507 2976 98648 83200 48116 21762 56520 53605 38841 82335 8328 60824 92567 33765 4771 6881 77520 94075 57381 8213 76810 99864 92593 14496 28180 55831 21746 51843 10756 61493 41208 94407 63595 52056 59215 64452 41064 81247 50455 99627 39643 4860 1227 51089 33753 96119 55374 53918 3998 76918 3734 9107 93029 6921 43349 93226 25044 83497 76807 45260 83422 88627 74570 89517 40048 86927 53106 65106 89819 32789 42040 46939 44576 69127 36045 60728 27816 41027 47490 7134 14669 77388 90893 58920 3704 1909 40384 83192 94523 86786 98012 19007 71099 65151 87108 7845 84223 70297 66614 12148 79158 73916 65413 88431 24094 78084 32562 81302 32547 70115 89950 32415 47919 66388 13343 54992 5190 26696 96626 56060 48015 73468 47801 74538 70877 69799 56326 52881 80546 86327 64579 11116 46398 23601 8899 3980 96112 51061 99240 91360 54895 71990 
208 5684 19216 27076 25382 7902 30655 48701 45818 7786 37605 18646 39321 43786 25182 4265 9227 6307 1804 48446 41352 32371 16009 12848 14050 28950 17331 38190 9312 13229 39962 44084 14446 26204 41950 38920 16409 16027 2302 48698 37182 44428 22262 46198 898 34540 21306 41557 24731 44163 12635 7656 500 30147 7802 7227 36640 48330 19714 3729 39034 14640 22489 48175 9909 33731 672 14500 49776 32555 2136 16966 43124 46479 12877 16298 6245 25002 12311 16113 3665 31310 10100 4451 42119 47602 47046 35088 15297 14014 14236 16693 5425 48761 36099 9238 11697 41475 47734 38735 36450 34489 40093 39802 21148 6438 25271 44575 26689 43233 14005 29406 34576 11304 30012 41680 19940 14132 14720 39659 37928 17501 17749 38921 47747 42930 5176 32558 33887 26454 41689 1052 12212 34693 29196 31479 38090 41665 25573 3189 24231 6547 24936 15695 24303 27727 39556 9795 28373 18463 29365 22073 20641 19241 18796 5690 3859 41746 7250 39256 34347 12875 34886 47421 12866 27402 16965 45525 24922 30537 14070 10888 49851 46128 45379 34385 30043 21015 23126 30298 24268 713 25079 42791 27874 32778 31108 33656 42673 2197 46408 25316 31968 21574 48300 22084 33542 44131 
200
11246 25308 98092 94193 14045 64626 53349 46678 2158 9722 34351 90550 61026 80472 78760 90836 13304 85384 97315 87969 23490 21353 81904 90648 89522 96502 93913 36170 81575 41462 99074 84706 30051 35382 91681 43272 94515 66587 73794 14453 42989 86052 79219 66397 73657 91456 31502 66128 97321 85453 17303 84692 58833 57806 76921 64721 15224 29221 87551 82433 79572 12235 1513 45312 82922 18686 93139 96769 98704 94359 53123 37894 25345 49793 732 83232 44187 27453 79591 90728 31239 1258 34459 92886 23302 94808 58867 41684 5171 87394 57377 32763 83886 9381 35892 32465 56471 13491 3233 51438 54539 52354 67901 71464 52964 94140 82307 58299 86567 42758 54511 70674 46012 80950 21601 48589 18572 7125 79704 56503 76323 72424 94990 15965 82218 59244 12488 63763 86466 43357 34052 87658 78628 52929 12733 20479 85450 64839 30491 59669 70662 73576 4144 36819 80530 87367 38519 38678 69431 23746 75017 20980 60350 34861 82364 73854 55989 9602 49534 42106 25053 15468 55275 33271 66130 34947 95257 75551 92063 23996 30382 70137 13801 75898 89368 64115 31280 39253 94179 873 12015 52206 93407 1167 72533 15617 47257 95726 27348 59078 86713 17875 42450 56762 82368 32511 61378 15835 42861 74764 
29387 6406 5683 33330 32999 40023 36953 26922 40476 11423 8583 37658 43403 42085 39544 39716 17281 26438 18095 4434 34338 13994 29426 43014 18067 13131 23261 48015 12912 5308 32558 37854 39840 24436 48385 46485 10485 1721 9887 21430 28655 17781 45119 48905 18198 48178 38622 8778 23126 31821 32476 34580 35191 21968 47472 5603 33656 47991 8879 46605 43174 24447 36639 40961 23205 48297 45625 40362 26321 27596 13340 22807 43087 30547 32028 7768 44148 49921 29030 5245 2680 5962 18227 6843 20702 48769 6987 37506 37670 46355 27894 4886 45020 19404 15425 23549 28993 12806 44847 1530 13440 32516 1646 16605 21895 2529 2251 30130 29012 47685 6833 3282 48220 29564 45537 42954 49668 5148 30236 6319 45792 47891 32922 34623 1747 44366 8077 35645 16039 14902 38983 32067 20335 29911 11881 33535 34579 33722 26338 42566 31619 49749 49472 47380 43023 9886 8034 36385 32277 47633 48806 31031 9289 21941 3093 7183 10789 42771 17565 42786 10388 4936 15847 16841 14404 47073 34260 9144 19263 11396 40346 43726 42496 29949 24727 11703 49713 1099 36627 7166 45976 4207 39775 2703 26612 48873 1744 20214 20976 14318 47164 24648 2223 34659 18371 48628 26908 33313 23092 38262 
196
36619 53628 24324 61056 94660 93545 14500 68789 75923 45927 53652 73803 93024 82032 61273 74152 23713 24704 56407 44502 57038 87302 26893 86739 56206 98136 99286 10693 64975 39145 21301 37485 88835 95400 38279 6049 81906 62453 8164 29242 93376 661 3005 74098 29859 52410 23700 55554 85655 70169 97905 62696 4893 78073 62703 83333 83848 11538 78166 99908 89185 14713 69594 19587 87781 17726 75244 64149 48819 55817 91714 49497 9509 11861 12290 94548 87064 74800 98099 99792 85382 69388 1990 49224 89191 9329 70563 28149 4954 40264 55243 17393 54761 43995 84665 38603 54113 74484 15095 20518 14728 77026 73101 2055 16293 34124 94886 94992 48556 25616 70475 52377 84513 49986 82765 806 4685 25438 9508 96810 14634 50500 19293 63953 65617 13283 64114 75949 19822 35940 49042 47274 85962 47554 17860 13619 49817 9047 80550 77687 91980 63202 7100 75199 52626 61020 46357 32222 33120 59434 12066 56997 58852 88645 73822 24013 66386 54651 44548 1708 505 52611 19120 31260 44022 72489 35627 49952 29775 92925 18792 40317 6311 29508 21706 10392 87485 59505 8549 36812 64192 88453 32212 87713 1645 88103 9300 32842 19778 52709 84042 98981 29833 60298 54366 77378 
18329 35595 1561 48133 31417 24306 4696 8969 8746 22345 46646 30127 35549 46462 1848 3439 10704 38275 18484 2116 28776 5214 45317 23741 33903 34565 37405 21157 26334 9455 30806 16513 47047 45375 31750 11408 30058 44109 18902 24438 32982 17443 189 46515 31063 11868 27086 43176 9427 1863 3504 43087 31231 13928 19761 24425 30659 5073 18679 35147 17932 49925 6503 17655 960 30959 10674 3500 46185 48346 41445 5656 46063 37749 25517 14295 18243 47818 5933 49337 28563 2448 19901 25248 43985 2724 49989 42200 47317 21267 35407 11190 1570 46941 20520 34849 26376 47139 39141 37799 39898 5908 26205 47065 43930 36275 2721 24775 19245 25956 22982 3582 15112 34800 3644 3384 45067 23961 43693 27018 2285 39972 9079 38173 14266 19729 42608 30104 639 33475 41398 9063 21321 37081 20527 41660 12921 34709 44693 12726 31412 2092 36262 6204 25400 36057 27 24653 45297 317 39616 28809 34297 35271 4977 14309 46130 49963 22194 43688 23172 41643 12015 47916 9852 13873 17295 21465 34862 39180 14292 39806 28320 10293 17947 32630 40892 32060 9135 23487 36819 7616 12685 28873 22757 21923 21128 26791 32761 42752 29032 45540 29251 21924 8010 11532 
197
42371 25506 60192 85987 46574 63376 45296 98745 70604 16153 44283 38126 81177 97917 38620 15006 92717 28689 39416 10900 22016 68833 22437 51255 54935 30263 13039 97508 90711 40953 72229 98217 10122 60936 88117 97720 49879 8753 42477 74511 34428 94111 27940 39551 53480 85258 37977 96297 10416 53214 25033 48910 2973 80115 55796 24226 91639 34297 87674 96581 35512 34775 85977 88545 79023 60528 61583 28333 4412 26581 48400 1588 87132 34230 16416 74098 79168 46862 64122 89613 51651 60238 38065 10855 89775 57059 31756 39773 31777 32437 8404 9610 14935 2940 54223 56263 48780 19274 66506 72265 8793 58225 84041 48515 17823 86425 85030 20546 79030 8438 60201 16751 29381 56311 46596 59847 22003 45850 77806 7381 13847 13879 99935 58023 57966 53327 90485 1940 40554 82989 80850 99140 38526 21916 83387 34669 905 51464 99451 46348 85190 48218 11762 84796 5731 93763 64492 49357 52257 77975 3347 89089 15401 56501 67217 48550 6605 67 59100 73641 26343 43284 28357 41223 93913 59020 34645 63036 97780 44811 72825 61355 82294 60821 29184 49990 23053 74080 92609 45876 85688 34080 37552 28844 69876 16729 59303 30327 19245 78750 78643 34107 23928 16583 18077 37053 88282 
24107 36112 31392 40135 5507 36950 48217 35194 17061 44098 9702 1736 13861 10389 43266 19837 40172 2903 19494 29002 47397 31246 2877 11620 335 21399 15233 27886 23747 35074 12397 25435 9352 8699 23181 42533 24425 20077 14576 9861 36394 1809 42501 20830 22463 33772 32909 36899 4533 1098 1078 40254 35452 26303 39647 6745 5913 15499 21033 17182 20081 13638 18181 29881 3136 21122 32185 40677 20364 30324 42334 28450 17851 29874 42046 40769 43907 5990 49183 27970 33235 29468 5480 47779 38030 46054 27932 28739 1840 33389 9190 46114 10901 15547 32069 44755 10489 8208 39622 8701 42829 5990 45856 20307 37845 37618 48522 22931 28727 44657 17114 18619 18166 18722 5314 12290 30167 40541 47240 13833 18621 3744 20584 46776 13304 21856 37715 34184 30232 9297 15687 34600 19247 38792 16340 17876 46557 15743 41090 25275 41601 31523 13452 39534 44728 37180 9055 30691 40319 47366 22233 49981 39903 32081 10238 7803 22312 1178 26681 16854 48744 27870 19553 30088 28773 40494 10550 41155 17285 4645 42701 2667 36179 26599 10106 8677 7928 17247 45504 8880 27983 8966 7255 22042 21076 19120 19033 39574 31554 15638 32623 44474 14844 29798 5881 47275 12999 
200
23532 10845 5575 13589 67817 14609 64219 40490 11667 43555 85286 1752 63450 23062 40375 84549 89096 44986 44576 95623 74013 65388 93915 61870 23563 13906 97776 4795 13906 99687 65766 7370 52685 9637 70946 791 79308 81369 56694 20215 99353 97836 63561 34196 58558 16408 13263 70772 24983 11225 22733 56799 95507 91920 17273 86659 25691 93672 49459 9317 77247 15934 97792 90071 49367 72737 64423 14178 46155 15241 32360 96356 78344 52434 50404 22001 64947 19702 59045 87507 54942 35412 85270 66554 61497 82751 55707 51892 67484 7269 49959 93661 59448 36321 54717 49359 6905 60435 99287 37117 50737 11334 27998 26617 16691 9566 61498 85418 38005 27105 74016 34382 49425 86035 83069 21524 64240 73967 85265 21789 57941 25606 57618 56864 55166 37162 39338 58614 71181 80502 3515 43337 93505 53440 22777 12807 31574 79421 32293 33343 77223 26492 51576 50073 31405 84117 55662 31345 72858 86947 26094 40804 53541 73788 45474 47107 8233 42251 17104 64688 71667 94926 26533 83437 82848 66070 13968 14314 51520 29746 47401 91085 67379 17084 58617 27469 4102 29232 24529 27087 46838 28372 90387 40373 96050 21038 13073 99576 13305 62369 32016 68668 52562 50985 61853 62233 55364 31336 85801 32944 
7821 21457 40308 25703 10767 26925 28277 44849 33524 24767 9607 18507 15886 17891 41876 27365 10836 16569 46447 11964 36456 39743 13160 2358 48878 59 45031 47521 29078 23329 13298 41486 8271 19534 39371 29764 8469 38109 6755 44339 20599 16179 6114 7434 48545 35630 27876 28486 38293 29158 29698 39999 30607 34699 23930 9058 7768 48671 34722 24167 33440 27278 23761 48142 18758 27704 32552 26003 19077 12896 7470 7500 43206 45816 27165 46397 44611 49401 45310 46828 17467 37075 13069 6287 36964 24339 14298 13777 7692 39351 45311 4535 42563 34884 44773 26396 37816 28491 40843 15562 34596 32071 29301 29250 41635 6887 18224 28291 11263 44163 6859 28365 13124 8002 18248 23717 38877 6009 5937 1128 17868 48775 32591 1152 23622 10083 6944 41850 1257 27859 27278 46492 15655 18510 3916 13501 3035 40883 7830 14924 29123 34831 42138 39398 16588 39304 35829 5153 46306 8100 15681 46354 47911 475 38214 12405 23799 10558 45401 27037 5918 31799 31781 4399 21495 26466 27128 25521 4953 12732 41448 33660 25719 20479 26244 16272 27091 45547 17150 12062 16069 35559 46220 29252 13922 38362 33507 21168 15085 11242 7984 13092 42426 30739 152 20102 12746 47745 24373 39574 
200
57582 57417 94982 31663 22446 14313 81756 81237 57206 49459 6172 32699 54999 34362 72107 84318 39993 99295 40523 75643 44998 78307 53498 3782 31485 93320 35040 68509 44308 17418 16571 22729 33183 46886 99719 53509 50037 63575 49677 76876 19897 34391 44315 95956 6936 16292 49711 74552 88436 65684 51371 67375 73663 89439 58984 26107 13716 24628 83642 11811 97496 56552 60894 83180 99158 66426 24369 44832 20100 56623 63044 73155 6787 28475 66191 72804 9342 17444 27380 17484 63023 17 2703 47382 34000 78299 70898 9751 613 58988 84602 40168 2290 47778 47103 16395 18135 7196 30547 24192 31526 81831 39964 49732 95923 43361 62306 33396 79928 51741 40333 6490 98439 67187 5516 66764 33110 41475 50006 4710 40593 17909 36639 68054 84588 37546 80784 20571 55644 80112 9738 55944 63465 78432 35904 90828 78364 41524 70666 60864 92275 36002 35119 53039 2854 84603 85521 3114 91092 23000 27187 47999 32909 9906 17023 22942 22800 55931 54512 58255 34791 9100 25235 86354 19861 62525 1751 79463 94346 37974 47327 28720 62398 49703 15869 22795 43718 55001 83119 12593 87318 86695 92615 20701 62125 54807 90011 47018 42805 88907 11698 62106 60629 41221 66844 14242 76490 38878 27423 34339 
18942 31890 21691 29375 22901 42326 21981 15787 36355 37233 10738 42648 630 1 25793 46295 37445 14708 22398 9370 15828 20731 22264 41896 11735 3811 7066 18144 1647 35025 41873 25085 27405 7436 35266 42306 39650 7123 30370 40058 37966 35324 25573 31362 10805 20521 34632 27504 30815 5406 11483 37485 18928 28971 22372 11791 14129 15208 17637 27294 30363 46877 19310 45549 11737 45978 9130 33189 49157 30277 15094 14408 37920 49952 30479 25092 42584 30692 475 6707 15656 14113 9750 34588 8668 14024 19781 33154 12194 15739 2015 23547 48687 8217 47141 48842 43175 642 12452 46180 18499 28012 34982 27440 42953 25974 16008 42493 36127 13636 17648 36005 13902 7071 43252 15881 10756 17975 41249 6331 36828 34056 37614 35884 36478 24582 36203 8110 13282 9047 14379 44057 11665 10346 44021 39220 3592 6856 18203 22540 3386 8686 3919 29893 30230 3942 30460 12074 10223 1539 49393 23434 8186 36039 47057 39865 37734 28398 34661 30946 15013 41552 10613 33394 19198 34203 47923 41465 32808 42273 27618 21472 20406 41032 42617 10822 38272 4065 17776 5600 29662 44385 16433 11342 43053 17574 6057 28585 21592 19018 39237 34262 43049 7491 42072 34903 31620 36134 25839 42822 
196
11267 67596 41780 24707 19609 38846 17175 96379 79704 2395 63319 18452 1594 69834 17297 20448 62690 5352 48025 71780 41670 38840 78705 64660 36263 58799 77230 22015 27985 35300 24682 32211 71961 78232 10573 4669 53954 52537 80101 24597 11030 67274 40970 70531 91729 54519 67534 44487 96716 62476 73870 73375 71485 15617 47065 4421 87994 63921 92333 87091 57333 82288 50207 80051 81742 17877 8588 67344 60312 21284 64280 7974 67698 9576 76060 32558 99127 51915 11683 63317 17810 75250 25076 86088 8101 87516 19990 52521 1717 27678 37051 45108 76928 62666 56685 64893 15612 14508 70030 77020 50183 24273 37072 94792 28791 64738 49793 71260 37707 43623 26244 29316 74707 83210 16339 87481 9041 7466 41948 3260 37006 95401 25103 72669 76099 74652 28811 80083 37240 15855 83341 83309 22054 65500 90631 51403 27628 53515 11807 76471 42821 3274 5787 47386 3124 75064 75365 30284 43160 51517 9911 96728 95851 27110 91672 93834 63946 60564 64295 40396 56078 82676 15494 47136 51295 86322 95678 27772 13903 71837 43294 13217 93921 14150 88989 38820 25769 55897 47935 87001 69662 77643 40623 22167 37823 95845 81576 91942 98711 48011 52891 3032 97968 66079 89382 18114 
20362 48660 32366 24057 36613 4311 18436 17012 3549 23372 35123 30670 37185 6058 32809 26044 47923 15821 7082 42526 36324 24715 24818 4564 34352 41554 33046 17801 8013 23507 23262 30979 26167 22522 24416 20686 42144 24144 3151 41500 25334 33635 43977 25586 5171 18897 6764 5260 35901 6906 46252 4315 47719 1896 21922 227 9178 11048 7866 9948 32139 41650 41988 8003 22728 39856 18232 36066 18063 13753 330 7635 42720 16588 4003 269 44784 45692 32078 26266 7782 20217 41202 39650 49898 7250 23777 45417 6900 13039 22020 19410 29313 8382 30571 32983 11514 46808 41834 6006 17219 5104 9722 37048 13049 18017 47399 33989 46091 25699 17157 2449 26783 22945 41001 30552 13639 22 46363 42160 9022 42591 25634 26150 1352 15356 10864 2768 47138 42617 1218 11811 9717 31608 1118 10745 18527 27499 6332 23218 45792 38852 365 40988 21629 7425 21273 23137 33212 45668 13565 22440 48726 28979 48787 14923 29419 33902 1579 37983 1418 43781 4057 6524 37407 19044 3919 8289 38542 29821 14699 20641 37072 41638 49742 25620 25885 37735 34202 27380 4557 20665 44483 17934 10633 26618 4020 43924 15325 10483 44918 37177 35151 24701 40815 36983 
200
65724 14960 61632 55340 42451 48513 37560 59138 33986 58754 42599 22415 42747 52531 76947 55215 97621 87413 9578 39731 97598 49675 63671 30320 79137 15877 45076 36184 38590 6251 50136 64061 9492 8231 40230 51945 56970 44797 40147 87214 28827 38929 31692 64459 30399 62445 72056 92524 51057 38067 33431 67265 9452 56918 48874 6058 84808 62743 13109 99370 78825 43163 47440 56818 84530 23777 58369 98026 26984 3724 15767 75796 4203 10056 58249 44050 93141 56669 44354 44640 87962 69024 57671 64207 26769 54649 83798 87953 49850 63775 6090 66411 67374 31881 94170 16898 12362 96448 94378 85011 40151 68710 33226 2210 38262 89650 53857 78151 35738 59583 88195 37087 231 18272 1591 95719 43720 23885 76530 85875 27823 80829 32700 59104 84433 77257 86119 45257 65343 5208 74049 27459 59312 60034 17228 50058 23843 82207 69842 96750 10437 51467 9249 46232 71774 74653 66414 50090 62430 32612 91058 1719 81936 57669 50779 84531 93805 67766 42213 91990 30531 78170 5103 46189 68761 57256 2130 91607 70179 1862 65501 82345 87541 28990 67406 30366 32313 79532 7562 32005 11515 32437 11123 41980 212 88918 31182 97256 34480 35983 30539 23584 80298 66574 52857 26371 33019 47891 9260 48281 
421 13835 25597 27290 1556 3951 30466 31140 23503 12678 1058 38018 1927 18909 42018 16379 22795 23058 20410 25970 6550 17022 33306 25569 36873 27091 40658 38773 42992 5277 6993 26555 7094 27692 8413 49319 7018 3528 22240 9021 4002 8926 48495 32000 12616 24883 35042 45173 42598 10816 15300 15458 35068 46295 8917 30075 44346 17984 48366 21175 16875 26249 45570 9250 46168 29307 3670 39747 6627 23659 26690 19008 1823 48584 5791 34959 47362 23763 29362 37510 20179 1082 37149 11034 1369 174 8238 48520 8986 14297 7667 30598 26366 44557 45459 20745 41477 14745 47950 43966 18668 14183 15546 2661 4253 16332 18498 40529 39647 14722 18091 20452 41072 32687 14215 43582 29250 19896 24043 4509 27040 23454 43091 42724 827 24516 38051 29498 16837 3513 9209 4554 12820 21846 11931 35705 24049 28224 18349 14092 49465 44172 27133 19386 42532 5810 13454 17096 47019 46669 8435 40330 33680 4602 43252 6727 16814 38068 36111 24339 27432 43376 8497 12419 13601 21322 26576 17633 3313 26874 13313 41682 46182 563 46483 39654 30443 14100 41077 12461 40889 16911 46259 48239 1683 25887 49910 34175 2271 33994 34867 15793 5007 49022 43421 8442 25155 29705 46720 21052 
199
4500 65688 1331 87070 75173 69696 91019 17951 83526 94196 3693 22771 68028 75378 54457 65000 97983 7107 55355 9700 2109 20730 90834 67551 74178 91673 32628 15265 79389 77028 35210 56262 269 33721 36986 83695 36075 50843 53009 71986 82890 17165 76021 89275 95910 97819 5034 39996 91968 39418 59610 68670 97362 86005 87267 76230 20721 62967 15310 19452 14117 73470 63122 87544 78442 47662 45009 73604 3294 76716 11676 74597 90718 23233 42009 17996 96496 12149 18724 47282 53666 21606 21645 66918 92606 87142 92224 14111 53449 25073 28270 61610 99216 18844 55027 95965 77591 55693 320 20114 68293 33687 30956 22222 25346 61316 84035 33452 3187 5467 99709 18481 31543 11746 93473 5346 90893 49799 77838 48002 54317 33409 29301 45042 15790 79054 19927 21343 26668 53473 28662 63397 12912 22845 55034 91545 80289 5143 61346 84967 64083 17420 60121 96548 97182 49822 41812 66786 11633 84944 98168 33375 64553 7726 89205 60055 78929 65783 12616 54336 2073 30404 44460 94477 95849 94422 15078 42871 34376 38083 94147 98754 10887 36788 36912 50930 30132 30055 87591 19355 94029 52680 23667 71503 7315 70674 253 42721 36971 8787 43775 98617 62149 49363 71776 56901 28331 18415 86361 
44174 6388 17185 16165 14120 36588 25696 36985 43751 28811 29271 28457 4418 26852 34764 46108 37211 11613 12643 27682 41795 3116 33780 9307 28979 27921 37209 47334 6530 31740 44200 40003 10708 44165 20815 21756 35135 41976 44038 7650 21000 22819 30617 18290 46213 44009 27917 12244 11211 48643 21409 15122 32528 30269 1107 13633 1025 45532 2802 27531 49618 34397 17588 10832 24949 49315 30679 27572 37171 2672 20635 7171 46142 13935 13319 28487 45282 48449 48722 11711 45029 28018 28015 8592 21595 26646 36218 18172 29638 2110 42736 47718 24852 16188 7486 36002 18360 16606 48281 4212 16908 35150 47649 23085 9630 26219 31635 46240 12061 26632 48000 26866 16011 18 22243 48040 4953 3232 31970 105 2853 21044 49547 30247 42000 157 28160 37848 384 22182 18913 39204 16516 38673 13819 38087 12617 4102 848 21459 7010 42997 29995 10394 35996 13204 1782 34433 28586 41338 2964 2457 21150 8462 3331 16425 46680 7951 28315 10451 27324 20113 30243 24910 10907 29712 40957 33476 42440 45111 36218 11685 14030 49539 26877 49471 40262 25787 11231 12533 43266 48830 37245 13459 28745 15703 34210 28607 7956 32961 1868 21333 13435 19307 19632 41257 14018 41997 20920 
