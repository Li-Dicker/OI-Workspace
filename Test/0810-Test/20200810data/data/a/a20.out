Case #1: 14244279
Case #2: 15374160
Case #3: 15681045
Case #4: 14027966
Case #5: 14221898
Case #6: 14379820
Case #7: 14126832
Case #8: 13800118
Case #9: 14181221
Case #10: 14443879
